ios/RNFrequency.m

```diff
@@ -1,13 +1,92 @@
-
 #import "RNFrequency.h"
+#import "TGSineWaveToneGenerator.h"
+
+// Used to send events to JS
+#if __has_include(<React/RCTBridge.h>)
+#import <React/RCTBridge.h>
+#elif __has_include("RCTBridge.h")
+#import "RCTBridge.h"
+#else
+#import "React/RCTBridge.h"
+#endif
+
+#if __has_include(<React/RCTEventDispatcher.h>)
+#import <React/RCTEventDispatcher.h>
+#elif __has_include("RCTEventDispatcher.h")
+#import "RCTEventDispatcher.h"
+#else
+#import "React/RCTEventDispatcher.h"
+#endif
+
+@interface RNFrequency ()
+@property(strong) TGSineWaveToneGenerator *toneGenRef;
+@end
 
 @implementation RNFrequency
 
+@synthesize bridge = _bridge;
+
+static NSString * const AUDIO_CHANGED_NOTIFICATION = @"AUDIO_CHANGED_NOTIFICATION";
+
+- (instancetype)init
+{
+    if (self = [super init]) {
+        self.toneGenRef = [[TGSineWaveToneGenerator alloc] initWithChannels:2];
```

Comment:
Is it common to have integers declared inline in Obj-C code, or should this either brought out into a constant or documented that it needs to be 2 in order to use both audio channels?